platform/ios/src/MGLMapView.h

```diff
@@ -286,6 +301,21 @@ MGL_EXPORT IB_DESIGNABLE
  */
 - (IBAction)showAttribution:(id)sender;
 
+/**
+ The preferred frame rate at which the map view is rendered.
+
+ The default value for this property is
+ `MGLMapViewPreferredFramesPerSecondDefault`, which will adaptively set the
```

Comment:
>  which will adaptively set the ...
does this means that it will be overridden by the sdk according to the device?

Reply:
Yes, `MGLMapViewPreferredFramesPerSecondDefault` means that the SDK will decide, based on the list of legacy devices in `UIDevice.mgl_isLegacyDevice`, [whether or not to set the FPS limit to `30` or `60`](https://github.com/mapbox/mapbox-gl-native/pull/12501/files#diff-20502b6bc02986a4398d0da84b2d4b94R1158).